cpp/src/arrow/filesystem/azurefs_test.cc

```diff
@@ -376,6 +376,10 @@ culpa qui officia deserunt mollit anim id est laborum.
     return s;
   }
 
+  std::string RandomFileName(RNG &rng) { return RandomChars(10, rng); }
```

Comment:
I can, yes.  Note that my use case will be to access ADLS, which is why I referred to this as a file.  Also because the method is named DeleteFile(), not DeleteBlob().  That said, I get the sense that it would be preferred to move these inline and delete these two functions.  So I'll go with that for my next update.